arviz/plots/__init__.py

```diff
@@ -1,15 +1,15 @@
 """Plotting functions."""
-from .autocorrplot import autocorrplot
-from .compareplot import compareplot
-from .densityplot import densityplot
-from .energyplot import energyplot
-from .forestplot import forestplot
-from .kdeplot import kdeplot, _fast_kde, _fast_kde_2d
-from .parallelplot import parallelplot
-from .posteriorplot import posteriorplot
-from .traceplot import traceplot
-from .pairplot import pairplot
-from .jointplot import jointplot
-from .khatplot import khatplot
-from .ppcplot import ppcplot
-from .violintraceplot import violintraceplot
+from .autocorrplot import plot_autocorr
+from .compareplot import plot_compare
+from .densityplot import plot_density
+from .energyplot import plot_energy
+from .forestplot import plot_forest
+from .kdeplot import plot_kde, _fast_kde, _fast_kde_2d
```

Comment:
Are these supposed to be private at the in __init__.py?

Reply:
Yes, I think they should be, because they are not meant for the average user.

I think they still propagate to main namespace?

Reply:
Yes but only because we import them into __init__.py I believe. I think if we don't import them here they won't propagate to the main namespace.

In either case I'm not too worried about it now and don't want to block this for something this minor, especially since "private" isn't even a real thing in Python.